homeassistant/components/flume/sensor.py

```diff
@@ -86,9 +129,29 @@ def state(self):
     @property
     def unit_of_measurement(self):
         """Return the unit the value is expressed in."""
-        return "gal"
+        # This is in gallons per SCAN_INTERVAL
+        return "gal/m"
+
+    @property
+    def available(self):
+        """Device is available."""
+        return self._available
+
+    @property
+    def unique_id(self):
+        """Device unique ID."""
+        return f"Flume {self._device_id}"
+
+    @property
+    def device_state_attributes(self):
+        """Return the device specific state attributes."""
+        return {"device_id": self._device_id}
```

Comment:
Is this really necessary ?

Reply:
Didn’t realize they can be disabled by the entity registry so no

Reply:
.. and they couldn't be disabled before because they had no unique id.   Got it

Reply:
removed